lib/graphql/schema/field.rb

```diff
@@ -642,12 +642,24 @@ def fetch_extra(extra_name, ctx)
       # @param field_ctx [GraphQL::Query::Context::FieldResolutionContext]
       # @return [Hash<Symbol => Any>]
       def to_ruby_args(obj, graphql_args, field_ctx)
+        @context = field_ctx.query.context
```

Comment:
Hmm, this field is a long-lived part of the schema -- it's created during initialization and stays around as long as the app is running. Also, this instance is shared between threads if the application is served by a multithreaded server. So I don't think caching query-specific values will work here.

Reply:
Honestly, I think you could just ignore this method. It's used by the legacy runtime, and in my opinion, we don't have to add features to that runtime. Alternatively, if supporting the legacy runtime is important for _you_, I won't stop you!

Reply:
I want to support as many runtimes as possible.
 I think if I create another module and move `load_application_object`, I can support this runtime.